pages/common/asdf.md

```diff
@@ -0,0 +1,28 @@
+# asdf
+
+> Command line interface for managing versions of different packages.
+> Homepage: <https://asdf-vm.com>.
+
+- List all available plugins:
+
+`asdf plugin-list-all`
+
+- Install a plugin:
+
+`asdf plugin-add <name>`
+
+- List all available versions for a package:
+
+`asdf list-all`
```

Comment:
Here the description says "for a package". Which package? Doesn't this command miss an argument?